subprojects/liburing.wrap

```diff
@@ -6,4 +6,4 @@ source_hash = 60b367dbdc6f2b0418a6e0cd203ee0049d9d629a36706fcf91dfb9428bae23c8
 patch_directory = liburing
 
 [provide]
-uring = uring
+dependency_names = liburing, uring
```

Comment:
This is just wrong, all wrong.

I don't know why it originally said uring, if upstream never used that name but did use liburing. We should ***not*** be exposing phony garbage names. At all.